### PR TITLE
No ticket - DIFI updated docs

### PR DESCRIPTION
**Pull request by EdPutans**

### CODE REVIEW

**Review by Luciam91 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,16 +66,98 @@ export default App
...
+  </Logging>
+</Config>
+```
+and a bunch more issues making our code impossible to maintain.
+
+So to alleviate that, we migrated to using a store solution - `Zustand` in this case.
+
+### How Zustand works
+
+Evertything we need for state management sits inside `src/beautifulSugar`.
```
Comment (AdenFraser) [inline README.md:94]: I didn't really understand why it's called `beautifulSugar` — can you provide context ?

File: README.md [hunk 2]
```diff
@@ -253,7 +367,9 @@ You can then embed the fingerprint script in any website using the following scr
 
 ## Testing
 
-## Manual Testing
+### Manual Testing
+
+#### [MAB] Please note that this is currently failing on their domains due to their security policy only allowing the prod version of DiFi. You can still test on localhost or on their stage-65 domains.
```
Comment (AdenFraser) [inline README.md:372]: Shouldn't call out a specific customer, since the repo is currently public.

File: README.md (hunk 1 above)
Comment (EdPutans) [reply README.md:94]: just a cool name i picked while being at 10KM above ground

File: README.md (hunk 2 above)
Comment (EdPutans) [reply README.md:372]: oh no

File: README.md (hunk 1 above)
Comment (AdenFraser) [reply README.md:94]: In future can you use ubiquitous language, so that others don't have to guess what things mean or what they are for :)

**Pull request merged**